packages/SwingSet/src/controller/startXSnap.js

```diff
@@ -82,10 +87,17 @@ export function makeStartXSnap(bundles, options) {
     // console.log('fresh xsnap', { snapStore: snapStore });
     const worker = doXSnap({ handleCommand, name, ...meterOpts, ...xsnapOpts });
 
+    const bundlePs = bundleIDs.map(id => bundleHandler.getBundle(id));
+    let bundles = await Promise.all(bundlePs);
+    if (overrideBundles) {
+      bundles = overrideBundles; // replace the usual bundles
```

Comment:
Should this include verification that the overrides are valid? Is such verification possible?

Reply:
not really.. one unit test overrides it with an empty list, that's really all it's used for